opds.py

```diff
@@ -611,27 +612,9 @@ def single_entry(cls, _db, work, annotator, force_create=False):
                                  force_create=force_create)
 
     @classmethod
-    def render_messages(cls, messages_by_urn):
```

Comment:
I've checked that `render_messages` is not used anywhere except in code that's removed below.